utils.py

```diff
@@ -12,6 +12,7 @@
 import json
 import os
 import logging
+import nh3
 from bs4 import BeautifulSoup
 
 def perform_notebook_conversion(notebook_json):
```

Comment:
I am curious about this utils.py file, it feels redundant to the convert_ipynb. and the convert_rmd files that you created separately, should this be deleted?

Reply:
Oh, this will be removed! Just need to finalize the PR before I delete it in case I need something :)